frontend/src/Component/History/QuickFilter.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+const QuickFilterButton = ({ label, onClick }) => (
```

Comment:
This should be a discussion with the group, but for any components like this. I feel like they should be in their own reusable file. Just to reduce code, as each file should do one thing and one thing only. But, that is also a group decision.